safe/gui/tools/test/test_wizard_dialog.py

```diff
@@ -876,8 +876,8 @@ def test_auto_select_one_item(self):
         message = 'It should auto select, but it does not.'
         self.assertTrue(dialog.lstSubcategories.currentRow() == 0, message)
         num_item = dialog.lstSubcategories.count()
-        message = 'There is should be only one item, I got %s' % num_item
-        self.assertTrue(num_item == 1, message)
+        message = 'There are should be only two items, I got %s' % num_item
```

Comment:
@timlinux 's point is we need to remove the `are` word. The previous `is` word is wrong also.